_articles/de/best-practices.md

```diff
@@ -179,6 +179,8 @@ Sie müssen nicht alles selbst machen. Die Gemeinschaft Ihres Projekts existiert
 
 Wenn Sie auf der Suche nach Mitwirkenden sind, fragen Sie doch einfach mal herum.
 
+Sie können auch [Issues, die für Anfänger einfach genug sind, explizit markieren](https://help.github.com/en/articles/helping-new-contributors-find-your-project-with-labels). GitHub weist auf solche Issues vermehrt hin, sodass beitragswillige Neulinge zu nützlicher Mitarbeit ermuntert werden.
```

Comment:
As usual, do we want to put `(Englisch)` here for links that aren't (yet) accessible in German?